src/Apps/Settings/Routes/MyCollection/MyCollectionRoute.tsx

```diff
@@ -81,22 +83,30 @@ const MyCollectionRoute: FC<MyCollectionRouteProps> = ({ me, relay }) => {
       {total > 0 ? (
         <>
           {!hasDismissedMessage && (
-            <Message
-              variant="info"
-              mt={-SETTINGS_ROUTE_TABS_MARGIN}
-              mb={SETTINGS_ROUTE_TABS_MARGIN}
-            >
-              <Flex flexDirection="row" justifyContent="space-between">
-                <Box />
-                <Text mx={1}>
-                  Access all the My Collection features on the Artsy app. Coming
-                  soon also on web.
-                </Text>
-                <Clickable onClick={dismissMyCollectionMessage}>
-                  <CloseIcon />
-                </Clickable>
-              </Flex>
-            </Message>
+            <FullBleed>
+              <Message
+                variant="info"
+                mt={-SETTINGS_ROUTE_TABS_MARGIN}
+                mb={SETTINGS_ROUTE_TABS_MARGIN}
+              >
+                <Flex flexDirection="row" justifyContent="space-between">
+                  <Box />
+                  <Link to="/settings/my-collection#download-app-banner">
+                    <Text mx={1}>
+                      Access all the My Collection features on the Artsy app.
+                      Coming soon also on web.
+                    </Text>
+                  </Link>
+
+                  {/**TODO: Replace Close with Clickable
+                   * if you can get clickable to not center its children
+                   * */}
+                  <Close onClick={dismissMyCollectionMessage}>
+                    <CloseIcon />
+                  </Close>
```

Comment:
What do you think about using the [Clickable](https://github.com/artsy/force/blob/b96e157ba9abfbe321c9b3f9ca198b7401f232a2/src/Components/Modal/Modal.tsx#L71) here?

Reply:
It was originally with Clickable it is almost impossible for the children not to center with this flex layout. https://github.com/artsy/force/pull/10648/files#diff-e8f31aad1a0c51e2e8ff1301402f54800d53bd23779054b4b696ad828351db43R101